scripts/ci/check_patch.sh

```diff
@@ -10,3 +10,7 @@ python setup.py checkdocs
 
 # stop the build if there are any formatting errors
 pre-commit run --all-files
+
+# check bash completio
+python -m dvc completion -v -o scripts/completion/dvc.bash
```

Comment:
Not sure if/how it runs right now, but deps are not installed yet, so you might very easily get an import error here.

Reply:
outdated - removed